heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -179,17 +179,24 @@ public boolean isEmpty() {
             final DataInMemoryReporter dataInMemoryReporter = reporter.newDataInMemoryReporter();
 
             final QuotaWatcher watcher = new QuotaWatcher(
-                options.getDataLimit().orElse(dataLimit).asLong().orElse(Long.MAX_VALUE), options
-                .getAggregationLimit()
-                .orElse(aggregationLimit)
-                .asLong()
-                .orElse(Long.MAX_VALUE), dataInMemoryReporter);
+                options.getDataLimit()
+                       .orElse(dataLimit)
+                       .asLong()
+                       .orElse(Long.MAX_VALUE),
+                options.getAggregationLimit()
+                       .orElse(aggregationLimit)
+                       .asLong()
+                       .orElse(Long.MAX_VALUE),
+                dataInMemoryReporter
+            );
```

Comment:
I agree that this looks better. However, the auto-formatting rules doesn't agree and will change this back the next time someone auto formats. The same is true for most of the other formatting changes in this file, except for the variable name changes (which I agree with).
So please either change the auto formatting rules somehow, or auto format before committing.